pkg/initialize/client.go

```diff
@@ -2,29 +2,50 @@ package initialize
 
 import (
 	"context"
+	"net/http"
 
 	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-02-01/storage"
 	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure/auth"
 
 	"github.com/openshift/openshift-azure/pkg/api"
 )
 
 type azureStorageClient struct {
-	accounts storage.AccountsClient
-	storage  azstorage.Client
+	accounts     storage.AccountsClient
+	storage      azstorage.Client
+	pluginConfig *api.PluginConfig
 }
 
-func newAzureClients(ctx context.Context, cs *api.OpenShiftManagedCluster) (*azureStorageClient, error) {
+func (az *azureStorageClient) addAcceptLanguages() autorest.PrepareDecorator {
```

Comment:
I suspect something like this is going to have to sit somewhere central (invent a pkg/util/azureclient or something) for now; later we probably need to refactor to get our azure clients from one place generally, but that requires more thought